lib/models/item_model.dart

```diff
@@ -0,0 +1,106 @@
+class ItemModel {
```

Comment:
Use a more descriptive name to this class. You can take it more specific like your info is relative to movies.

Reply:
Changed!